### PR TITLE
Added queries and data

### PR DESCRIPTION
**Pull request by gjuliao**
Last animal seen by William Tatcher
image

Count animals that visited Stephanie Mendez
image

List all vets specialties, even those with no specialties
image

Animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
image

Animal with most vets visits
image

Maisy Smith's first visit?
image

Most recent visit: animal information, vet information, and date of visit.
image

Visits with vet not specialize in animal's species
image

Look for Masy Smith most frequent specialty.
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -31,10 +31,34 @@ name varchar(100),
...
-CREATE TABLE VETS (
+CREATE TABLE vets (
 id SERIAL,
 name varchar(100),
 age INT,
 date_of_graduation DATE,
 PRIMARY KEY (id)
+);
+
+CREATE TABLE specialties (
```
Comment (Ghiftee) [inline schema.sql]: - [ ] According to the requirements, the name of this table should be ```specializations``` let's try to use the same name here.

File: schema.sql [hunk 2]
```diff
@@ -31,10 +31,34 @@ name varchar(100),
...
+        REFERENCES species (id)
+CONSTRAINT vets_id
+    FOREIGN KEY (vets_id)
+        REFERENCES vets(id)
+);
+
+CREATE TABLE visits (
+id SERIAL,
+animals_id INT,
+vets_id INT,
```
Comment (Ghiftee) [inline schema.sql:57]: - [ ] Good job adding the various columns to this table, however, you are missing the ```date``` column, kindly add it

File: data.sql [hunk 3]
```diff
@@ -31,4 +31,38 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
...
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 3, '2020-07-22');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (4, 4, '2021-02-02');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-01-05');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-03-08');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (1, 2, '2020-05-14');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (6, 3, '2021-05-04');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (7, 4, '2021-02-24');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2019-12-21');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 1, '2020-08-10');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (9, 2, '2020-04-07');
```
Comment (Ghiftee) [inline data.sql:57]: - [ ] Kindly change this date to ```2021-04-07``` as this would help you have more accurate data.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @gjuliao 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ New tables are created
✔️ Required data were entered into the table
✔️ README is professional
✔️ Query screenshots were added

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [ ] Great job adding query screenshots to your PR, however, you also need to run the queries in the ```Queries.sql``` file.
- [ ] Kindly fix your MIT license as it is currently broken. A fixed license would help your project be more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @gjuliao 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the required changes from the previous reviewer.

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**